app/controllers/transactions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class TransactionsController < ApplicationController
+  include ListingsHelper
```

Comment:
This is sad.
There is two other possibilities: 
- moving translate_quantity_unit function to ApplicationHelper (it is used in other functions of ListingsHelper) or 
- we duplicate the function to TransactionsHelper / TransactionsController